lib/commanded/commands/router.ex

```diff
@@ -263,8 +277,11 @@ defmodule Commanded.Commands.Router do
 
   """
   defmacro middleware(middleware_module) do
+    middleware_module = :elixir_aliases.expand_or_concat(middleware_module, __CALLER__)
```

Comment:
Where does the `:elixir_aliases.expand_or_concat/2` function come from, I can't find it mentioned in the Elixir docs and is it safe to take a dependency on it?

Reply:
It's used in Macro.expand, see https://github.com/elixir-lang/elixir/blob/v1.14.2/lib/elixir/lib/macro.ex#L1771

I guess it's a private api 😅 but I did not find another way to get the full module name without introducing a compile-time dependency on it.

Reply:
I don't think we want to take a dependency on an internal (private) function.